Weather Man/Weather_Man/FileReader.py

```diff
@@ -0,0 +1,59 @@
+class FileReader():
+    def __init__(self):
+        self.pkt = []
+        self.max_temperature_c = []
+        self.mean_temperature_c = []
+        self.min_temperature_c = []
+        self.dew_point_c = []
+        self.mean_dewpoint_c = []
+        self.min_dewpoint_c = []
+        self.max_humidity = []
+        self.mean_humidity = []
+        self.min_humidity = []
+        self.max_sea_level_pressureh_pa = []
+        self.mean_sea_level_pressureh_pa = []
+        self.min_sea_level_pressureh_pa = []
+        self.max_visibility_km = []
+        self.mean_visibility_km = []
+        self.min_visibility_km = []
+        self.max_wind_speed_km_h = []
+        self.mean_wind_speed_km_h = []
+        self.max_gust_speed_km_h = []
+        self.precipitation_mm = []
+        self.cloud_cover = []
+        self.events = []
+        self.wind_dir_degrees = []
+
+    def data_filler(self, file_path_list=[]):
+        i = 0
+        while(i < len(file_path_list)):
```

Comment:
Why use while when and track counter manually when you can use for loop instead?

```py
for file_path in file_path_list:
```

Also rename `file_path_list` to `file_paths`. Plural name means its a list.